app/components/MadeBySwapr.tsx

```diff
@@ -0,0 +1,86 @@
+'use client';
+
+import { useState, useEffect } from 'react';
+
+const cities = ['Stockholm', 'Porto', 'Barcelona'];
+
+const getRandomCityIndex = () => Math.floor(Math.random() * cities.length);
+
+export const MadeBySwapr = () => {
+  const [currentCity, setCurrentCity] = useState(cities[getRandomCityIndex()]);
+  const [isAnimating, setIsAnimating] = useState(false);
+
+  useEffect(() => {
+    const getRandomCity = () => {
+      const currentIndex = cities.indexOf(currentCity);
+      let newIndex;
+      do {
+        newIndex = getRandomCityIndex();
+      } while (newIndex === currentIndex);
+
+      return cities[newIndex];
+    };
+
+    const updateCity = () => {
+      setIsAnimating(true);
+
+      setTimeout(() => {
+        setCurrentCity(getRandomCity());
+        setIsAnimating(false);
+      }, 200);
+    };
+
+    const DURATION_IN_MS = 10000; // 10 sec
```

Comment:
Why not call the variable `_10_SECONDS`, `TEN_SECONDS` or `_10_SECONDS_IN_MS` and delete the comment?